vehicles.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE IF NOT EXISTS `player_vehicles` (
+    `id` int(11) NOT NULL AUTO_INCREMENT,
+    `license` varchar(50) DEFAULT NULL,
+    `citizenid` varchar(50) DEFAULT NULL,
```

Comment:
Should this be a required field? Or does it need to be null for backwards compatibility?

Reply:
I'm not sure as to whether it needs to be null. I can't really think of a time or reason where it would need to be anything other than filled with a players citizenId

Reply:
QBCore had it as nullable but uses real values every time it inserts or updates values. Probably an oversight.

Reply:
So it should just be required?

Reply:
Seems like it, yes

Reply:
changed license and citizenId